data/categories.txt

```diff
@@ -14863,6 +14863,51 @@ zh-Hant:國家公園
 fa:پارک ملی
 mr:राष्ट्रीय उद्यान
 
+boundary-aboriginal_lands
+# For English, "American Indian" is used by at least some Indigenous people in the US to self describe but
+# the standalone term "Indian" should be avoided for confusion with people from India.
+# Likewise the term "Aboriginal" is used at least in the context of Australia and it's
+# level of acceptability varies depending on which Indigenous person/group you ask.
+# See: https://github.com/organicmaps/organicmaps/pull/7186#discussion_r1460107442
+en:Indigenous Lands|Aboriginal Lands|9American Indian Reservations|7First Nations Reserve
```

Comment:
A number like 5 or greater before Indigenous can be helpful.

Shorter synonyms are preferred, nobody will type longer strings consisting of many words in the search. E.g. aboriginal|indian|reservations

Question: can these territories be considered by some tourists as sightseeing places?

Reply:
> A number like 5 or greater before Indigenous can be helpful.

True

> Shorter synonyms are preferred, nobody will type longer strings consisting of many words in the search. E.g. aboriginal|indian|reservation

I object to "indian" on its own being a synonym as it is easily confused with people or places from the country of "India" (also has more negative connotations compared to any of the other terms). For the rest I can work with that

> Question: can these territories be considered by some tourists as sightseeing places?

Not really, the reservations are (some) places where Indigenous folks live. They don't necessarily have remarkable features for tourists to look at